.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,73 @@
+name: Devtron-auto-labeller
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  manage-labels:
+    if: ${{ !github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Authenticate with GitHub CLI
```

Comment:
You can reduce this step by using - https://github.com/devtron-labs/devtron/blob/main/.github/workflows/create-release.yml#L65